lib/Controller/PageController.php

```diff
@@ -84,7 +84,7 @@ public function index(?string $hash = null): TemplateResponse {
 	 */
 	#[NoAdminRequired()]
 	#[NoCSRFRequired()]
-	#[FrontpageRoute(verb: 'GET', url: '/{hash}/{view}')]
+	#[FrontpageRoute(verb: 'GET', url: '/{hash}/{view}', requirements: ['hash' => '[a-zA-Z0-9]{16}'])]
```

Comment:
Maybe to be safe for the future
```suggestion
	#[FrontpageRoute(verb: 'GET', url: '/{hash}/{view}', requirements: ['hash' => '[a-zA-Z0-9]{16,}'])]
```